tekton/utils/pipeline.go

```diff
@@ -57,7 +57,8 @@ type Pipeline struct {
 // the Pipeline.
 // +kubebuilder:object:generate=true
 type ParameterizedPipeline struct {
-	Pipeline
+	Pipeline `json:",inline"`
+	// +optional
```

Comment:
Missing empty line before this one and missing documentation.

Reply:
added.